pkg/scaffold/v2/internal/string_utils.go

```diff
@@ -115,3 +111,36 @@ func InsertStringsInFile(path string, markerAndValues ...string) error {
 
 	return err
 }
+
+// filterExistingValues removes the marker,value pair for values that already
+// exists in the given reader.
+func filterExistingValues(r io.Reader, markerAndValues ...string) (map[string][]string, error) {
```

Comment:
IIRC each item in `markerAndValues` must NOT be a multi-line string.
Please add a comment for it.

Reply:
Done.